todo_app/templates/index.html

```diff
@@ -12,7 +12,20 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.id }} - {{ item.status }} - {{ item.title }}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Create Item</h2>
+      <form action="/items" method="post">
+        <label for="title">Enter item title: </label>
+        <input type="text" name="title" id="title" required>
```

Comment:
Nice use of `required` here, worth noting that this validation is only currently on the frontend so users could bypass it if they chose to, but I think in this situation that's quite reasonable

Reply:
Sure, thanks.